rules-tests/Transform/Rector/Attribute/AttributeKeyToClassConstFetchRector/Fixture/not_replacing_existing_constant.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Fixture;
+
+use Doctrine\ORM\Mapping\Column;
+use Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Constants\Constant;
+
+class SomeClass
+{
+    #[Column(type: Constant::VALUE)]
+    public $name;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Fixture;
+
+use Doctrine\ORM\Mapping\Column;
+use Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Constants\Constant;
+
+class SomeClass
+{
+    #[Column(type: Constant::VALUE)]
+    public $name;
+}
+
+?>
```

Comment:
if before vs after is equal, then after part is not needed:

```suggestion
```

Reply:
Removed 👍